### DISCUSSION TITLE
New preview release

### DISCUSSION THREAD

**Discussion by jvalkeal** [openpgpjs/openpgpjs#1290] [General]
Would it be possible to get a new preview release v5.0.0-2?

Been planning to start upgrading from v4 to v5 in some of my own stuff and what's in master works for me as there's some needed typescript changes. Would be really helpful to get a new preview into npmjs.

**Comment by twiss** [comment]
Hey :wave: Sorry for the delay. Here you go: https://github.com/openpgpjs/openpgpjs/releases/tag/v5.0.0-2.
